tests/test_skills/test_apy_estimation_abci/test_apis.py

```diff
@@ -97,7 +102,44 @@ def test_eth_price(spooky_specs: SpecsType, eth_price_usd_q: str) -> None:
         res = make_request(api.get_spec(), eth_price_usd_q)
         eth_price = ast.literal_eval(api.process_response(DummyMessage(res.content))[0]["ethPrice"])  # type: ignore
 
-        assert isinstance(eth_price, float)
+        assert eth_price == 0.4183383786296383
+
+    @staticmethod
+    def test_eth_price_non_indexed_block(
+        spooky_specs: SpecsType, eth_price_usd_q: str
+    ) -> None:
+        """Test SpookySwap's eth price request from subgraph, when the requesting block has not been indexed yet."""
+        spooky_specs["response_key"] = "errors"
+        api = SpookySwapSubgraph(**spooky_specs)
+
+        # replace the block number with a huge one, so that we get a not indexed error
+        current_number = "3830367"
+        largest_acceptable_number = "2147483647"
```

Comment:
Maybe use constants here as well

Reply:
Done in 65763b23c6c6c3d5c2f4ab662999c8d303602b72.